src/backers/admin.py

```diff
@@ -25,6 +23,24 @@ def get_queryset(self, request):
             .annotate_aids_count(Backer.instructed_aids, 'nb_instructed_aids')
         return qs
 
+    def get_readonly_fields(self, request, obj=None):
+        fields = ('display_related_aids',)
+        # When we are on editing mode, we want the slug field to be
+        # readonly. When obj is there, is means we are on editing mode.
+        if obj:
+            return fields + ('slug',)
```

Comment:
Suggestion : n'utiliser qu'une seule instruction `return` par fonction.

```suggestion
            fields += ['slug']
```